prime-router/src/main/kotlin/transport/EmailTransport.kt

```diff
@@ -0,0 +1,75 @@
+package gov.cdc.prime.router.transport
+
+import com.sendgrid.*
+import com.sendgrid.helpers.mail.Mail
+import com.sendgrid.helpers.mail.objects.*
+import gov.cdc.prime.router.EmailTransportType
+import gov.cdc.prime.router.TransportType
+import gov.cdc.prime.router.azure.DatabaseAccess
+import org.thymeleaf.TemplateEngine
+import org.thymeleaf.context.Context
+import org.thymeleaf.templateresolver.StringTemplateResolver
+import java.nio.file.Files
+import java.nio.file.Path
+import java.util.Calendar
+
+class EmailTransport : ITransport {
+
+    override fun send(
+        orgName: String,
+        transportType: TransportType,
+        header: DatabaseAccess.Header,
+        contents: ByteArray
+    ): Boolean {
+
```

Comment:
I think you will need to rebase with master. ITransport has changed.

Reply:
Rebased